static/panes/gccdump-view.ts

```diff
@@ -120,13 +122,14 @@ export class GccDump extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Gcc
         this.eventHub.emit('gccDumpUIInit', this.compilerInfo.compilerId);
     }
 
-    override initializeCompilerInfo(state: Record<string, any>) {
+    override initializeCompilerInfo(state: PaneState) {
         super.initializeCompilerInfo(state);
 
-        if (!state.id && state._compilerid) this.compilerInfo.compilerId = state._compilerid;
-        if (!state.editorid && state._editorid) this.compilerInfo.editorId = state._editorid;
-        if (!state.compilerName && state._compilerName) this.compilerInfo.compilerName = state._compilerName;
-        if (!state.treeid && state._treeid) state.treeId = state._treeid;
+        if (!state.id && (state as any)._compilerid) this.compilerInfo.compilerId = (state as any)._compilerid;
+        if (!state.editorid && (state as any)._editorid) this.compilerInfo.editorId = (state as any)._editorid;
+        if (!state.compilerName && (state as any)._compilerName)
+            this.compilerInfo.compilerName = (state as any)._compilerName;
+        if (!state.treeid && (state as any)._treeid) state.treeid = (state as any)._treeid;
```

Comment:
this is the only thing I'm unsure on, renamed `state.treeId` to `state.treeid` but as far as I can tell it's not read anywhere